### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Bhargavijanga58**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,63 +1,126 @@
...
-1. Make it time affordable for less-coder scientists (who know nothing about WfMS) to manage their data processing workflows
+1. Make it time affordable for scientists with less-coding experience (who are unware of WfMS) to manage their data processing workflows.
 
-2. Preserve all the model run history and share them along with the code files
+  
 
-# [Installation](install.md)
+2. Preserve every model run history and share them along with the code files.
 
-# [Tutorial](https://zihengsun.github.io/Geoweaver/)
```
Comment (ZihengSun) [inline docs/index.md:23]: why remove this link?

**Review by ZihengSun — CHANGES_REQUESTED** [review]
look good and minor comment

File: docs/index.md (hunk 1 above)
Comment (Bhargavijanga58) [reply docs/index.md:23]: As it is redirecting to the same webpage, below we have links to follow up on the tutorial, that is the reason I made it as a heading. Kindly let me know if I am wrong.

File: docs/host.md [hunk 2]
```diff
@@ -1,92 +1,355 @@
...
 
+  
+
+  
+
+  
+
 ## What is Host?
 
+
```
Comment (ZihengSun) [inline docs/host.md:18]: why adding so many empty lines?

File: docs/index.md (hunk 1 above)
Comment (ZihengSun) [reply docs/index.md:23]: Actually not really. This markdown is only available in GitHub which is not the same link. The link `https://zihengsun.github.io/Geoweaver/` is generated using mkdocs and published in another branch.

Comment (Bhargavijanga58) [reply docs/index.md:23]: Okay professor. I will undo that change.

File: docs/host.md (hunk 2 above)
Comment (Bhargavijanga58) [reply docs/host.md]: I was using a software to edit and it added spaces. I will make sure not to repeat the same.

File: docs/install.md [hunk 3]
```diff
@@ -5,51 +5,48 @@
...
-* Step 2: Pull geoweaver image using command: 
+**Step 2**: Pull geoweaver image using command:
 
 `docker pull jensensun/geoweaver`
 
-* Step 3: Run geoweaver using command:
+**Step 3**: Run geoweaver using command:
 
-`docker run -t -i -v <YOUR_HOME_DIRECTORY>:/home/marsvegan/ -p 8070:8070 jensensun/geoweaver`
+    `docker run -t -i -v <YOUR_HOME_DIRECTORY>:/home/marsvegan/ -p 8070:8070 jensensun/geoweaver`
```
Comment (ZihengSun) [inline docs/install.md:38]: this space makes the command not rendered correctly in GitHub. can you double check in Github after every commit? make sure it appears the same as in your local software.

File: docs/install.md [hunk 4]
```diff
@@ -5,51 +5,48 @@
...
 
-> *What is going on?* : `-v <YOUR_HOME_DIRECTORY>:/home/marsvegan/` is for mounting your current home directory into the docker containers. `marsvegan` is the user name within geoweaver containers. `-p 8070:8070` maps the port so you can access Geoweaver from your browser. `jensensun/geoweaver` is the published docker image url in DockerHub.
+>  *What is going on?* : `-v <YOUR_HOME_DIRECTORY>:/home/marsvegan/` is for mounting your current home directory into the docker containers. `marsvegan` is the user name within geoweaver containers. `-p 8070:8070` maps the port so you can access Geoweaver from your browser. `jensensun/geoweaver` is the published docker image url in DockerHub.
 
-> *Tip*: You can create an alias so next time you won't copy and paste such a long command:
+*Tip*: You can create an alias so next time you won't copy and paste such a long command:
 
-`alias geoweaver="docker run -t -i -v <YOUR_HOME_DIRECTORY>:/home/marsvegan/ -p 8070:8070 jensensun/geoweaver"`
+    `alias geoweaver="docker run -t -i -v <YOUR_HOME_DIRECTORY>:/home/marsvegan/ -p 8070:8070 jensensun/geoweaver"`
```
Comment (ZihengSun) [inline docs/install.md:44]: same here. can you just change markdown directly in text? don't use third party software. The auto formatting is a bit annoying.

File: docs/host.md [hunk 5]
```diff
@@ -5,77 +5,85 @@
...
 
-1. Select the new host under `Hosts`>`Linux/Win/Mac Computers` on the right panel.
+1. Select the New Host under `Hosts`>`Linux/Win/Mac Computers` on the left panel.
 
 2. Click the python icon button in the toolbar of the shown `Host Details` page.
 
 3. Input the `Host User Password` of your account on the remote server in the shown dialog pop up.
 
-> `Note`: if the selected server is localhost, password instructions are discussed [here](#install.md)
+>  `Note`: if the selected server is localhost, password instructions are discussed [here](#install.md)
```
Comment (ZihengSun) [inline docs/host.md:42]: this link doesn't work for some reason, double check

File: docs/host.md [hunk 6]
```diff
@@ -5,77 +5,85 @@
...
 2. Click the uploader icon button in the `Host Details` toolbar.
 
 3. Input the `Host Password` of your account on the server in the shown dialog pop up.
 
-> `Note`: if the selected server is localhost, password instructions are discussed [here](#install.md)
+>  `Note`: if the selected server is localhost, password instructions are discussed [here](#install.md)
 
 4. A file upload section will appear below the `Host Details` section with drag and drop or file browser capability.
+
 ![Screen Shot 2022-01-15 at 12 39 12 AM](https://user-images.githubusercontent.com/34227630/149588755-f1982927-a2b0-453f-8248-65fc5668428c.png)
```
Comment (ZihengSun) [inline docs/host.md:71]: We intend to NOT include screenshots in the tutorial, as our GUI is changing all the time. Also the link seems to be temporary and not sure if it is a permanent link. Please remove.

**Pull request merged**